dhis-2/dhis-services/dhis-service-audit-consumer/src/main/java/org/hisp/dhis/audit/legacy/TrackerAuditConsumer.java

```diff
@@ -52,26 +53,45 @@
 public class TrackerAuditConsumer implements AuditConsumer
 {
     private final AuditService auditService;
-    private final RenderService renderService;
+    private final ObjectMapper objectMapper;
+    private final boolean isAuditLogEnabled;
+    private final boolean isAuditDatabaseEnabled;
 
     public TrackerAuditConsumer(
-        AuditService auditService, RenderService renderService )
+        AuditService auditService,
+        ObjectMapper objectMapper,
+        DhisConfigurationProvider dhisConfig )
     {
         this.auditService = auditService;
-        this.renderService = renderService;
+        this.objectMapper = objectMapper;
+
+        this.isAuditLogEnabled = Objects.equals( dhisConfig.getProperty( ConfigurationKey.AUDIT_LOGGER ), "off" );
```

Comment:
Move the default value to `ConfigurationKey#defaultValue`. This way we can centralize it.